packages/start-server-core/tests/createStartHandler.test.ts

```
@@ -0,0 +1,81 @@
+import { describe, it, expect, beforeEach } from 'vitest'
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)